src/components/CustomTable/CustomizedTablePagination/CustomizedTablePagination.js

```diff
@@ -0,0 +1,38 @@
+import { TableFooter, TablePagination, TableRow } from "@material-ui/core"
+
+const CustomizedTablePagination = ({
+  rowsPerPageOptions,
+  totalRowsCount,
+  rowsPerPage,
+  tablePage,
+  setTablePage,
+  setRowsPerPage,
+  handleFetchMore,
+}) => {
+  const handleChangePage = (event, newPage) => {
+    handleFetchMore()
+    setTablePage(newPage)
+  }
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(parseInt(event.target.value))
+    setTablePage(0)
+  }
```

Comment:
You should be allowing your parent components to decide what happens in these functions